data/actionsmap/yunohost.yml

```diff
@@ -97,15 +97,6 @@ user:
                         pattern: &pattern_lastname
                             - !!str ^([^\W\d_]{2,30}[ ,.'-]{0,3})+$
                             - "pattern_lastname"
-                -m:
-                    full: --mail
```

Comment:
Too many apps and other misc script using that option in the wild, dropping it will make many things explode, so instead I'll implement a legacy behavior/deprecation...